src/test/java/net/snowflake/client/pooling/LogicalConnectionLatestIT.java

```diff
@@ -370,6 +376,77 @@ public void testDatabaseMetaData() throws SQLException {
     }
   }
 
+  @Test
+  public void testExceptions() throws SQLException {
```

Comment:
what exactly we are testing there? there are so many mocks that it is hard to understand if we are testing the production code even

Reply:
According to the logical implementation, almost all the methods have the try-catch block, and the catch blocks are not tested. However, it was not easy to make testing when the physical connection throws an exception with a regular instance. As a result, I made the physical connection throw an error and tried to test all the catch blocks of all the methods in the logical connection.

As you mentioned, I needed to edit the testing name. I will fix it.